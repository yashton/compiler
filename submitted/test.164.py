(d = "1.4\
 + 8}\
 hello there"