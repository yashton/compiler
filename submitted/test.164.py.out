(PUNCT "(")
(ID "d")
(PUNCT "=")
(ERROR "Illegal character '"1.4\
 + 8}\
 hello there"' on line 1)
