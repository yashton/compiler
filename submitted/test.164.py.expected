(PUNCT "(")
(ID "d")
(PUNCT "=")
(LIT "1.4 + 8} hello there")
(ENDMARKER)
